tokens/0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874.yaml

```diff
@@ -0,0 +1,20 @@
+---
+addr: '0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874'
+decimals: 18
+description: >- Tokenized 1 Troy Ounce Gold Bar, Redeemable in Precious Metal (Gold)
+  # The following options were found                                                                       #
+  # You may need to edit them and you MUST delete commented lines                                          #
+  # From the website: description meta tag                                                                 #
+  Veritaseum enables software-driven P2P capital markets without brokerages, banks or traditional exchanges
+links:
+- Facebook: https://www.facebook.com/reggiemiddletonfintech/?fref=ts
+- Telegram: https://t.me/joinchat/G_22OhA7UgN6b-7hTSFSTA
+- Twitter: https://twitter.com/reggiemiddleton
+- Website: https://veritas.veritaseum.com/
+- YouTube: https://www.youtube.com/channel/UCjVEF7ABJwyzw_8pb5frVmg
+- Blog: https://veritas.veritaseum.com/ve-blog
+- Reddit: https://www.reddit.com/r/VeritaseumCommunity/
```

Comment:
Is there really no white paper for this token?

Reply:
No white paper.
They do have a working product that operates on the network.
ICO wasn't launched as an investment opportunity, rather the tokens are a prepaid fee.
Earlier version ran on BTC blockhcain, but it was halted due to regulatory uncertainty.
So the secret herbs and spices are not available to see, but we see the VeADIR in action every time an exposure is opened.

Reply:
Would an introduction on the workings on the company's own website suffice to clearly illustrate what the token does?
The website has ample information on the company.
Also the team is quite impressive.
I have spoken to several personally, if that counts to anything.

Reply:
Just making sure we link it if one exists.